lib/dogma.ex

```diff
@@ -6,20 +6,29 @@ defmodule Dogma do
   other modules through the divine `run/3` function.
   """
 
-  alias Dogma.Formatter
   alias Dogma.Rules
   alias Dogma.ScriptSources
 
   @version Mix.Project.config[:version]
 
-  def run(dir, config, formatter) do
+  def run(dir, config, events) do
```

Comment:
The formatters (or whatever we call them) are really the listeners here, this is more of a dispatcher.  How about `event_manager` or just `dispatcher`. It's referred to as an event manager in the docs. I'm partial to `dispatcher` but that's less explicit.